lib/ui/viewer/file/video_widget_new.dart

```diff
@@ -63,6 +66,11 @@ class _VideoWidgetNewState extends State<VideoWidgetNew> {
         }
       });
     }
+    player.stream.playing.listen((event) {
```

Comment:
Is there any need to dispose anything?

Reply:
Couldn't find anything to dispose. We are disposing the player though.

Reply:
Yes, have disposed a listener and a valueNotifier.